### DISCUSSION TITLE
Request Labels

### DISCUSSION THREAD

**Discussion by whiskeysquared** [sct/overseerr#3914] [Feature Requests]
Ability for admin, or user with permissions to add a label, or bulk add labels to requests to note any details about the request for future reference. 

Use case: I like to invent collections. I'll find a "thing" to form the collection around (actor, director, trope, etc) and go nuts on Overseerr requesting (basically creating a list for myself to go download later, I only have Overseerr not the rest of the Rr ecosystem, so I do things manually). I'll set those downloads up and sometimes it'll be weeks before it completes, and I have NO idea WHY I downloaded it, or what collection I thought to put it in. It would be great to be able to go to Overseerr and find the label/note/memo I added when I requested it.

I couldn't find a request exactly like this. My use of the word "labels" isn't to be conflated with Plex labels.

**Comment by Lockszmith-GH** [comment]
I'm not a maintainer, just a user like you. (We'll almost, I do run the other systems in the background).

Not sure Overseerr is the right tool for that. Don't get me wrong - I'd love that, but there are tools that already do that proficiently well.
+ Overseer relies on the ability of the arr system behind it to do that.

Letterboxd or trackt.tv would probably be better for that at this point.

You can install Sonarr and Radarr quite easily without using it's automation, they have tags, which you can select in overseerr.
They are actually tools that were designed for what you're looking for.

I'm sorry if I seem obtuse, if I'm missing something obvious to you, please explain the reasonoing.
